README.md

```diff
@@ -31,6 +35,62 @@ Alternatively, when using from Lagom:
 libraryDependencies += "com.lightbend" %% "lagom-service-locator-dns" % "0.1.0"
 ```
 
-## Motivation
+Your platform will require differing "transformers" that will translate a service name
+into a DNS SRV name to be looked up. Here is the Typesafe config declaration to be 
+considered:
+
+```
+service-locator-dns {
```

Comment:
Does the user need to always provide transformers in its application or do we have a `reference.conf` for that in this library?

My previous comment in regards to default configuration was this:

> Wondering if we should use a DC/OS, Mesos, Marathon translator as the default. This library should be agnostic to such things.

> What about: Documenting possible translators in the reference.conf for Marathon, Kubernetes and Consul. Additionally, we could provide a sbt settings key to set a list of translators. For this settings key we could also provide default entries for marathon, kubernetes and consul.

> Not sure though how the sbt settings key can be leveraged by maven.